go-controller/pkg/ovn/namespace.go

```diff
@@ -218,7 +227,23 @@ func (oc *Controller) updateNamespace(old, newer *kapi.Namespace) {
 	}
 	defer nsInfo.Unlock()
 
-	annotation := newer.Annotations[hotypes.HybridOverlayExternalGw]
+	var annotation string
+	//var parsedAnnotation net.IP
+	annotation = newer.Annotations[routingExternalGWsAnnotation]
+	if annotation != old.Annotations[routingExternalGWsAnnotation] {
+		// TODO: handle list of IPs
+		if annotation == "" {
+			for k, v := range nsInfo.ecmpRoutes {
+				_, stderr, err := util.RunOVNNbctl("--if-exists", "--policy=src-ip",
+					"lr-route-del", v.gr, v.podIP+"/32", k)
```

Comment:
is this the way the command should be formated? from the ovn-nbctl docs 
 `[--if-exists] lr-route-del router [prefix]` 

so shouldn't it be 

`util.RunOVNNbctl("--if-exists", "--policy=src-ip","lr-route-del", v.gr, v.podIP+"/32")`

I could have read something wrong

Reply:
Currently it works, but I'll double check, it may be I'm adding additional args that I need